pkg/handler/handler.go

```diff
@@ -149,6 +149,7 @@ func (o *OpenAPIService) RegisterOpenAPIVersionedService(servePath string, handl
 	}{
 		{"application", "json", o.getSwaggerBytes},
 		{"application", "com.github.proto-openapi.spec.v2@v1.0+protobuf", o.getSwaggerPbBytes},
+		{"application", "com.github.proto-openapi.spec.v2.v1.0+protobuf", o.getSwaggerPbBytes},
```

Comment:
We're re-using these strings, especially the invalid one in multiple places, can we use a constant?

Reply:
used constants in handler/handler.go